pages/developers/20-21.tsx

```diff
@@ -40,7 +40,7 @@ const developers = ({ devWing, coHead }: DeveloperProps) => {
 
 export const getStaticProps: GetStaticProps = async () => {
   const response = await fetch(
-    "https://css-website.herokuapp.com/api/admin/members/20-21"
+    "https://tasty-crab-hosiery.cyclic.app/api/admin/members/20-21"
```

Comment:
Add the base URL as an environment variable.....and replace it here and everywhere
else with that env variable.

1. Create a file .env.local in root folder
2. add a variable BACKEND_URL = https://tasty-crab-hosiery.cyclic.app

Then use it everywhere like: `${process.env.BACKEND_URL}/api/admin/endpointName`